simple_history/models.py

```diff
@@ -221,6 +221,32 @@ def get_instance(self):
                 for field in fields.values()
             })
 
+        def get_next_record(self):
+            """
+            Get the next history record for the instance. `None` if last.
+            """
+            entry = self.instance
+            history = entry.history.order_by('-history_date')
+            hist_list = [hist for hist in history]
+            try:
+                entry_hist_idx = hist_list.index(self)
+            except ValueError:
+                return None
+            return hist_list[entry_hist_idx - 1] if entry_hist_idx > 0 else None
+
+        def get_prev_record(self):
+            """
+            Get the previous history record for the instance. `None` if first.
+            """
+            entry = self.instance
```

Comment:
I don't love going from historical record to instance back to historical record. Ideally we'd avoid this. Maybe look to see what they do in `cleanerversion`? https://github.com/rossmechanic/cleanerversion/blob/master/versions/models.py#L83

I think something similar to that on our historical tables would be a much cleaner solution